scripts/install_third_party_libs.py

```diff
@@ -167,6 +167,29 @@ def compile_protobuf_files(proto_files_paths: List[str]) -> None:
                 r'^import (\w*_pb2 as)', r'from proto_files import \1')
 
 
+def fix_google_module(google_module_path: str) -> None:
```

Comment:
The current function name `fix_google_module` is vague in terms of what we're trying to fix. Are we trying to fix all issues that involve google modules? If not, then I suggest renaming the function that addresses the aforementioned concern. From what I've read in the docs and the linked NDB issue thread, a better name would be `make_google_module_importable_by_python`.

Reply:
done!